pkg/driver/node.go

```diff
@@ -140,6 +148,17 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				// Fall through; the code below will uniq for us.
 			}
 
+			if f == "tls" {
+				klog.Warning(
+					"Use of 'tls' under mountOptions is deprecated with this driver. " +
+						"Set encrypt in transit in the volumeContext instead, e.g. 'encryptInTransit: true'")
```

Comment:
How about 

`Use of 'tls' under mountOptions is deprecated with this driver since tls is enabled by default`.

Reply:
fixed. I still mention encryptInTransit to disable .